extensions/oidc-client-reactive-filter/runtime/src/main/java/io/quarkus/oidc/client/reactive/filter/OidcClientRequestReactiveFilter.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.oidc.client.reactive.filter;
+
+import java.util.function.Consumer;
+
+import javax.annotation.Priority;
+import javax.inject.Singleton;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestContext;
+import org.jboss.resteasy.reactive.client.spi.ResteasyReactiveClientRequestFilter;
+
+import io.quarkus.oidc.client.Tokens;
+import io.quarkus.oidc.client.runtime.AbstractTokensProducer;
+import io.quarkus.oidc.client.runtime.DisabledOidcClientException;
+import io.quarkus.oidc.common.runtime.OidcConstants;
+
+@Provider
+@Singleton
```

Comment:
Is this needed?

Reply:
Just removed locally, all is fine without it